README.md

```diff
@@ -48,4 +48,95 @@ From within the root directory:
 1. Change the database username and password in these 2 files to reflect your setup.
 1. npm run installAll
 1. npm run initialize
-```
\ No newline at end of file
+```
```

Comment:
I would probably add a little description for each of the end points other than just the code. Would be something super quick and nice to add to the readme. Might be helpful for future use or even just good practice.